sdk/textanalytics/cognitiveservices-textanalytics/src/detectLanguageResultCollection.ts

```diff
@@ -14,14 +14,20 @@ import {
 } from "./detectLanguageResult";
 import { sortByPreviousIdOrder } from "./util";
 
+/**
+ * Collection of `DetectLanguageResult` objects corresponding to a batch of input documents, and
+ * annotated with information about the batch operation.
+ */
 export interface DetectLanguageResultCollection extends Array<DetectLanguageResult> {
   /**
-   * (Optional) if showStats=true was specified in the request this field will contain information
-   * about the request payload.
+   * Gets statistics about the input document batch and how it was processed
+   * by the service. This property will have a value when showStats is set to true
```

Comment:
nit: just noticed that here it's `showStats` (as in the API) but .net has this as `IncludeStatistics`. Just want to make sure this was intentionally on either side.

Reply:
We changed this in .NET to be more consistent with .NET naming conventions.  @xirzec , if it's not too out of alignment with JS standards, it would be good to be consistent across the languages here.